db/migrate/20170301192129_create_provider_tags_table.rb

```diff
@@ -0,0 +1,13 @@
+class CreateProviderTagsTable < ActiveRecord::Migration[5.0]
+  def change
+    create_table :provider_tags do |t|
```

Comment:
@blomquisg As a general rule, the FK from the tag to the resource is the ems_ref, so I'm not sure that we would need to add it explicitly.